web/html/src/branding/css/base/cveaudit.less

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+#accordion {
```

Comment:
I'm not convinced this is a good idea, this way we end up having a lot of very specific styles that are only used in this specific view. The overrides here are very specific and that makes them fragile. If the underlying panel styles change, something here will surely get out of alignment as well.

Reply:
The accordion was removed altogether so I'm resolving this.